packages/flutter/lib/src/animation/scroll_behavior.dart

```diff
@@ -105,26 +105,25 @@ class UnboundedBehavior extends ExtentScrollBehavior {
 Simulation _createFlingScrollSimulation(double position, double velocity, double minScrollOffset, double maxScrollOffset) {
   double startVelocity = velocity * _kSecondsPerMillisecond;
 
-  // Assume that we're rendering at atleast 15 FPS. Stop when we're
-  // scrolling less than one logical pixel per frame. We're essentially
-  // normalizing by the devicePixelRatio so that the threshold has the
-  // same effect independent of the device's pixel density.
-  double endVelocity = 15.0 * ui.window.devicePixelRatio;
+  // Assume that we're rendering at >= 15 FPS. Stop when we're
+  // scrolling less than one physical pixel per frame, which isn't
+  // visually detectable.
+  double endVelocity = 15.0 / ui.window.devicePixelRatio;
```

Comment:
Fair enough.

In any case I think it should be written in terms of time, not frequency. We care about the time it takes for the scroll bar to fade out, not the number of frames. If we want it to be 66ms, then it'd be `1 / (0.066 * ui.window.devicePixelRatio) // logical pixels per second)`. Or if you want the dimension analysis to be more obvious, `(1 / ui.window.devicePixelRatio) / (0.066)` maybe.